sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java

```diff
@@ -166,6 +166,18 @@ Flux<ServiceBusMessageContext> receive() {
         return receiveFlux;
     }
 
+    /**
+     * Gets a stream of stream messages from each session.
+     *
+     * @return A Flux of stream messages which is from each session.
+     */
+    Flux<Flux<ServiceBusMessageContext>> receiveFromRollingSessions(){
```

Comment:
This is one possible solution to solve the issue that we don't concurrently process messages from different sessions:
I add `receiveFromRollingSessions()` to return each stream of session messages `Flux<Flux<Message>>`, instead of using `Flux.merge` to merge all session messages into one Flux, so that in processor client, we can subscribe for each session independently and parallelly run them based on the number of `maxConcurrentCalls`.

This is the draft version to make sure I am in the correct direction. I am willing to hear more comments for the solutions.